zipline/algorithm.py

```diff
@@ -320,6 +321,7 @@ def __init__(self, *args, **kwargs):
         if not self.blotter:
             self.blotter = Blotter(
                 data_frequency=self.data_frequency,
+                equity_slippage=FixedBasisPointsSlippage(),
```

Comment:
How were we setting the default slippage model before if we weren't passing it here?  Should we be doing this there instead?

Reply:
it was set in blotter before, changed the default to be set in blotter, but made it configurable from `TradingAlgorithm`